### PR TITLE
[17.09] Bump containerd to 210bf85fcd2f277d2e7b1eb142dd748fbd69a946

### PR DESCRIPTION
**Pull request by thaJeztah**
Includes fixes for:

- https://github.com/moby/moby/issues/34683 Exec orphans are not reaped correctly (PR: https://github.com/containerd/containerd/pull/1445 )
- https://github.com/containerd/containerd/pull/1661 File descriptors not being closed: "Set FD_CLOEXEC for memory event fd"
- https://github.com/moby/moby/issues/35331 / https://github.com/containerd/containerd/issues/1699 Containerd unable to start in case of corrupted events.log (PR: https://github.com/containerd/containerd/pull/1720)

Full diff: https://github.com/containerd/containerd/compare/06b9cb35161009dcb7123345749fef02f7cea8e0...210bf85fcd2f277d2e7b1eb142dd748fbd69a946

Fixes https://github.com/moby/moby/issues/34683
Fixes https://github.com/moby/moby/issues/35331

### CODE REVIEW

File: components/engine/vendor/github.com/containerd/containerd/runtime/container_linux.go [hunk 1]
```diff
@@ -176,7 +177,7 @@ func (c *container) getMemoryEventFD(root string) (*oom, error) {
 		return nil, err
 	}
 	defer f.Close()
-	fd, _, serr := syscall.RawSyscall(syscall.SYS_EVENTFD2, 0, syscall.FD_CLOEXEC, 0)
+	fd, _, serr := syscall.RawSyscall(syscall.SYS_EVENTFD2, 0, unix.EFD_CLOEXEC, 0)
```
Comment (thaJeztah) [inline components/engine/vendor/github.com/containerd/containerd/runtime/container_linux.go]: (OT) Is this fix included in containerd 1.0 as well? The code changed, so this file is no longer there, so just double-checking

**Comment by thaJeztah** [comment]
ping @stevvooe @crosbymichael @mlaventure PTAL

**Comment by thaJeztah** [comment]
```
23:43:10 [DockerTrustSuite] --- FAIL: Test (7.91s)
23:43:10 [DockerTrustSuite] 	protect.go:75: Failed to load frozen images: ApplyLayer exit status 1 stdout:  stderr: unexpected EOF
23:
```

Not sure what's going on there; possibly there's fixes for that upstream? Or just a random node failure?

File: components/engine/vendor/github.com/containerd/containerd/runtime/container_linux.go (hunk 1 above)
Comment (mlaventure) [reply components/engine/vendor/github.com/containerd/containerd/runtime/container_linux.go]: It moved to the containerd/cgroups repo and the current tip includes the fix

**Comment by mlaventure** [comment]
The frozen images loading doesn't involve containerd AFAIK

File: components/engine/vendor/github.com/containerd/containerd/runtime/container_linux.go (hunk 1 above)
Comment (mlaventure) [reply components/engine/vendor/github.com/containerd/containerd/runtime/container_linux.go]: Oh, that's a 0.2.x version :)

**Comment by thaJeztah** [comment]
Same error on https://github.com/docker/docker-ce/pull/334#issuecomment-348581035, so doesn't look like related to this PR (also green now)

**Pull request closed without merging**